### PR TITLE
docs: Add initial nomad-driver-virt driver plugin documentation.

### PR DESCRIPTION
**Pull request by jrasell**
Jira: https://hashicorp.atlassian.net/browse/NET-11215

Deploy preview https://nomad-git-f-net-11215-hashicorp.vercel.app/nomad/plugins/drivers/virt

Fixes: [CE-680]

[CE-680]: https://hashicorp.atlassian.net/browse/CE-680?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/content/plugins/drivers/virt/client.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      uri      = "qemu:///system"
+      user     = ""
+      password = ""
+    }
+  }
+}
+```
+
+- `data_dir` - (default: `"/var/lib/virt"`) - a directory where VM configuration files and
+  intermediate files are stored.
```
Comment (aimeeu) [inline website/content/plugins/drivers/virt/client.mdx:66]:
```suggestion
- `data_dir` - (default: `"/var/lib/virt"`) - the directory for storing VM configuration files and
  intermediate files.
```

File: website/content/plugins/drivers/virt/client.mdx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+- `image_paths` - (default: `["/var/lib/virt", "${ALLOC_DIR}"]`) - an allow-list of paths the driver
+  is allowed to load an image from.
+
+- `emulator` - (block)
+  - `uri` - (default: `"qemu:///system"`) - The `uri` specifies which hypervisor to connect to.
+  - `user` - (default: `""`) - the username to use for authentication.
+  - `password` - (default: `""`) - the password to use for authentication.
+
+## Client Attributes
```
Comment (aimeeu) [inline website/content/plugins/drivers/virt/client.mdx:76]:
```suggestion
## Client attributes
```

File: website/content/plugins/drivers/virt/client.mdx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+When installed, the `virt` plugin provides the following node attributes which can be used as
```
Comment (aimeeu) [inline website/content/plugins/drivers/virt/client.mdx:78]:
```suggestion
When installed, the `virt` plugin provides the following node attributes that you can use as
```

**Review by aimeeu — APPROVED** [review]
Thanks James! I left style guide suggestions.

File: website/data/docs-nav-data.json [hunk 4]
```diff
@@ -1973,6 +1973,27 @@
         "title": "Raw Fork/Exec",
         "path": "drivers/raw_exec"
       },
+      {
+        "title": "Virt",
```
Comment (aimeeu) [inline website/data/docs-nav-data.json:1977]:
```suggestion
        "title": "Virt <sup>Beta</sup>",
```

Comment (aimeeu) [reply website/data/docs-nav-data.json:1977]: Add the beta designation (took me a while to find)

File: website/data/plugins-nav-data.json [hunk 5]
```diff
@@ -14,6 +14,27 @@
         "title": "Podman",
         "path": "drivers/podman"
       },
+      {
+        "title": "Virt",
```
Comment (aimeeu) [inline website/data/plugins-nav-data.json:18]:
```suggestion
        "title": "Virt <sup>Beta</sup>",
```

**Review by aimeeu — APPROVED** [review]
Add beta to Virt in the nav

File: website/content/plugins/drivers/virt/index.mdx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+layout: docs
+page_title: 'Drivers: virt'
+description: >-
+  The virt task driver leverages Linux capabilities and libvirt to run virtual
+  machines.
+---
+
+# Virt Task Driver
+
```
Comment (aimeeu) [inline website/content/plugins/drivers/virt/index.mdx:10]: I think we should include a Beta warning here.
```suggestion

~> Beta Feature: The Virt task driver is in beta. Do not use this driver in production environments.

```

File: website/content/plugins/drivers/virt/task-config.mdx [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+- `primary_disk_size` - (int: required) - Disk space to assign to the VM. Must fit the VM's OS.
+
+Example:
+```hcl
+config {
+  image                           = "local/focal-server-cloudimg-amd64.img"
+  use_thing_copy                  = false
+  hostname                        = ""
+  cmds                            = ["python -m http.server 8000"]
+  default_user_password           = "password"
```
Comment (schmichael) [inline website/content/plugins/drivers/virt/task-config.mdx:46]: Security scanners are going to ping us on this one. Mind commenting it out and/or making it a more obvious "do not actually use this" value?

```suggestion
  #default_user_password           = "CHANGE ME"
```

File: website/content/plugins/drivers/virt/task-config.mdx [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+      artifact {
+        source      = "http://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
+        destination = "local/focal-server-cloudimg-amd64.img"
+        mode        = "file"
+      }
+
+      config {
+        image                           = "local/focal-server-cloudimg-amd64.img"
+        primary_disk_size               = 26000
+        default_user_password           = "L5w&KVf6nTvp]~Z?>8}tj("
```
Comment (schmichael) [inline website/content/plugins/drivers/virt/task-config.mdx:133]: As above: security scanners are going to ping us on this one. Mind removing it (or commenting it out and making it a more obvious "do not actually use this" value)?

```suggestion
```

**Pull request merged**